<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>libface Example</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="configLocation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>441</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Some Default File Location</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openConfigBtn">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>505</y>
      <width>132</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Config Directory</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>100</y>
      <width>160</width>
      <height>168</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="openImageBtn">
       <property name="text">
        <string>Open Image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="detectFacesBtn">
       <property name="text">
        <string>Detect Faces</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="recogniseBtn">
       <property name="text">
        <string>Recognise</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateDatabaseBtn">
       <property name="text">
        <string>Update Database</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveConfigBtn">
       <property name="text">
        <string>Save Config</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>621</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
